classes/tl_form.php

```diff
@@ -76,25 +77,59 @@ public function sendFormNotification($arrData, $arrForm, $arrFiles, $arrLabels)
      */
     public function generateTokens(array $arrData, array $arrForm, array $arrFiles, array $arrLabels, $delimiter)
     {
-        $arrTokens = array();
-        $arrTokens['raw_data'] = '';
+        $arrTokens                  = [];
+        $arrTokens['raw_data']      = '';
```

Comment:
Please do not change any lines of code unrelated to this PR. You can make a separate code style draft PR, if you want.

Reply:
fixed